examples/sdk/react/src/App.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import './App.css';
+import { BacktraceClient } from '@backtrace/react';
+
+function App() {
+    const [clicked, setClicked] = useState(false);
+    function sendError() {
+        const client = BacktraceClient.instance;
+        if (client) {
```

Comment:
I think it's important to make it loud the client is not available for any reason. Can we throw an error?

Reply:
We throw an error in the instance getter if the client is not available.